flink/src/main/scala/org/apache/mahout/flinkbindings/FlinkEngine.scala

```diff
@@ -357,12 +357,12 @@ object FlinkEngine extends DistributedEngine {
     res.collect().head
   }
 
-  private def generateTypeInformation[K: ClassTag]: TypeInformation[K] = {
+  def generateTypeInformation[K: ClassTag]: TypeInformation[K] = {
     val tag = implicitly[ClassTag[K]]
```

Comment:
Thanks, I probably should have closed this since i already merged it,  I'll reopen another with for the list in the description. and make the above referenced fix.